.github/workflows/publish-php-debug-images.yml

```diff
@@ -6,6 +6,8 @@ on:
   schedule:
     - cron: "0 0 * * 0"
   workflow_dispatch:
+permissions:
+  contents: read
```

Comment:
we'd better check this one - "publish image" seems like it would need more permissions than this